torch/_dynamo/backends/distributed.py

```diff
@@ -445,10 +461,10 @@ def run_node(self, n: Node) -> Any:
                     # Finally, we have to produce inputs for use compiling the next submodule,
                     # and these need to be FakeTensors, so we execute the module under fake_mode
                     with fake_mode:
```

Comment:
Do we really need to keep this? We no longer actually propagate any fake_args information.